xml/transactional_updates.xml

```diff
@@ -0,0 +1,517 @@
+     continue to use <package>ntp</package>, a fresh installation is required
+     to work correctly with transactional updates.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <package>sblim-sfcb</package>: The whole sblim ecosystem is incompatible
+     with transactional update.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     <command>btrfs-defrag</command> from the
+     <package>btrfsmaintenance</package> package does not work with a read-only
+     root filesystem.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     For <command>btrfs-balance</command>, the variable
+     <literal>BTRFS_BALANCE_MOUNTPOINTS</literal> in
+     <filename>/etc/sysconfig/btrfsmaintenance</filename> must be changed from
+     <literal>/</literal> to <literal>/.snapshots</literal>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     For <command>btrfs-scrub</command>, the variable
+     <literal>BTRFS_SCRUB_MOUNTPOINTS</literal> in
+     <filename>/etc/sysconfig/btrfsmaintenance</filename> must be changed from
+     <literal>/</literal> to <literal>/.snapshots</literal>.
+    </para>
+   </listitem>
+  </itemizedlist>
+ </sect1>
+ <sect1 xml:id="sec.install-tu">
+  <title>Installing <package>transactional-update</package></title>
+
+  <para>
+   The <package>transactional-update</package> package is on &suse; Package
+   Hub, <link xlink:href="https://packagehub.suse.com/"/>. &suse; Package Hub
+   is available as a free extension to all users with a valid &productname;
+   subscription. You can enable the extension during installation, or
+   afterwards with &yast;, and then install the
+   <package>transactional-update</package> package like any other package with
+   Zypper or &yast;.
+  </para>
+
+  <para>
+   You may also elect to install
+   <package>transactional-update-zypp-config</package>, which prevents Zypper
+   from accidentally removing <package>transaction-update</package>.
+  </para>
+ </sect1>
+ <sect1>
+  <title>The <command>transactional-update</command> Command</title>
+
+  <important>
+   <title>Only use when requested by SUSE Support</title>
+   <para>
+    This reference for <command>transactional-update</command> should only be
+    used when requested by SUSE Support.
+   </para>
+  </important>
+
+  <para>
+   The <command>transactional-update</command> command enables you to install
+   or remove updates of your system in an atomic way. Updates are applied only
+   if all of them can be successfully installed.
+   <command>transactional-update</command> creates a snapshot of your system
+   before the update is applied, and you can restore this snapshot if the
+   update fails.
+  </para>
+
+  <para>
+   All changes become active only after reboot.
+  </para>
+
+  <para>
+   If the current root file system is identical to the active root file system
+   (after applying updates and reboot), remove all old snapshots:
```

Comment:
The snapshots will not be removed! This command will tell snapper, that this snapshots can be deleted on the next cleanup phase. From the documentation:

           If the current root filesystem is identical to the active root
           filesystem (means after a reboot, before transactional-update
           creates a new snapshot with updates), all old snapshots without a
           cleanup algorithm get a cleanup algorithm set. This is to make
           sure, that old snapshots will be deleted by snapper. See the
           section about cleanup algorithms in snapper(8).

           Also removes all unreferenced (and thus unused) /etc overlay
           directories in /var/lib/overlay.